refs/w3c.json

```
@@ -7414,6 +7417,12 @@
         "edDraft": "https://w3c.github.io/PNG-spec/",
         "repository": "https://github.com/w3c/PNG-spec"
     },
+    "PNG-19961001": {
+        "aliasOf": "png-1-19961001"
+    },
+    "PNG-20031110": {
+        "aliasOf": "png-1-20031110"
+    },
```

I think the main issue is a mix of uppercase/lowercase shortnames and a missing version"

```suggestion
    "PNG-19961001": {
        "aliasOf": "PNG-1-19961001"
    },
    "PNG-20030520": {
        "aliasOf": "PNG-1-20030520"
    },
    "PNG-20031110": {
        "aliasOf": "PNG-1-20031110"
    },
```